.github/dependabot.yml

```
@@ -1,8 +1,27 @@
 version: 2
 updates:
-  - package-ecosystem: "github-actions"
-    directory: "/"
+  - package-ecosystem: github-actions
+    directory: /
     schedule:
-      interval: "daily"
+      interval: weekly
     reviewers:
-      - "maintainers"
+      - amimart
+      - ccamel
+      - fredericvilcot
+      - ErikssonJoakim
+    assignees:
+      - ccamel
+      - fredericvilcot
+    open-pull-requests-limit: 5
+  - package-ecosystem: npm
+    directory: /
+    schedule:
+      interval: weekly
+    reviewers:
+      - ccamel
+      - fredericvilcot
+      - ErikssonJoakim
+    assignees:
+      - ccamel
+      - fredericvilcot
+    open-pull-requests-limit: 5
```

As you have added a `Dockerfile`, it should also be handled by `Dependabot`. (see: [dependabot#package-ecosystem](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#package-ecosystem)).

```suggestion
    open-pull-requests-limit: 5
  - package-ecosystem: docker
    directory: /
    schedule:
      interval: weekly
    open-pull-requests-limit: 2
    reviewers:
      - ccamel
      - fredericvilcot
      - ErikssonJoakim
    assignees:
      - ccamel
      - fredericvilcot
```